### DISCUSSION TITLE
ESET virus notification

### DISCUSSION THREAD

**Discussion by Noarifumirimota** [jgraph/drawio-desktop#1368] [General]
Today ESET has identified the application (no-installer ver.) as a Trojan virus. Has anyone encountered a similar problem?

**Comment by LighthouseFR** [comment]
Yes. ESET Endpoint Security shows Trojan Horse.
I downloaded a clean installer, checked installer with ESET before installation, result same.
Manually updated virus definition in ESET, checked the installer again, result: clean / no viruses detected

**Comment by ec-nstone** [comment]
I have had the same issue come up today as well.
and now the application is having some issue.  Tool bars not working, and some components are not displaying properly.

Detection
Action
User
Information
Hash
First seen here
6/29/2023 7:50:51 AM Real-time file system protection file
C:\Program Files\WindowsApps\draw.io.draw. ioDiagrams_21.5.0.0_x64_1zh33159kp73c\app\ resources.pak
JS/PSW.Discord.BT trojan
cleaned by deleting

Event occurred during an attempt to access the file by the application: C:\Program Files\WindowsApps\ draw.io.draw.ioDiagrams_21.5.0.0_x64_1zh33159kp 73c\app\draw.io.exe (2A853AC8C5B1B3F0FD809B 854005084CAC41EF42).
4B2B4E61A8105DD9583C12E9ADFD307C113907C8 6/26/2023 8:02:38 fevl

**Comment by davidjgraph** [comment]
Maybe someone could find the site for submitting false positives and submit the app?

**Reply by LighthouseFR** [reply]
No need, the false positive has been resolved after the auto virus definition update. I did a full (deep) scan on my workstation this morning with draw.io installed. I downloaded the installer did a scan on it. Result: virus free!
But feel free to submit a false positive.
https://support.eset.com/en/kb141-submit-a-virus-website-or-potential-false-positive-sample-to-the-eset-lab

**Comment by davidjgraph** [comment]
I'll close this up. Ultimately, this is a free app and these are commercial anti-virus tools. We can't spend a lot of time chasing every false positive on every anti-virus. We don't willingly introduce viruses and don't add external code without an audit, it can be safely assumed these are false positives.
